Docs/source/install/hpc/cori.rst

```diff
@@ -32,29 +32,8 @@ KNL
 
 We use the following modules and environments on the system (``$HOME/knl_warpx.profile``).
 
```

Comment:
This is a good idea, to have the scripts already set up and ready to run. However, the scripts are not mentioned in this documentation (only included). It would be worth stating that the following lines are already available in the script and give the path to it.

Reply:
Ok, I'll add captions with a hint where the example can be copied from.